### ISSUE TITLE
TypeError: sequence index must be integer, not 'str'

### ISSUE THREAD

**Issue by asibilia** [traversc/getPokeStats#4]
Hey, I'm getting the following error when running the script: 

```
Traceback (most recent call last):
  File "getPokeStats.py", line 26, in <module>
    items = response['responses']['GET_INVENTORY']['inventory_delta']['inventory_items']
TypeError: sequence index must be integer, not 'str'
```

Any idea what's going on? Thanks in advance!

**Comment by mwest-im** [comment]
It seems the response is empty - it may have to do with the updated API. I'm not the most familiar with what changes  need to happen (or have time to dig into it), but I imagine it's a small tweak to fix things.

**Comment by mwest-im** [comment]
I've looked into this more. I found two things.

First, you'll want to uninstall the git+https://github.com/tejado/pgoapi.git package and install the git+https://github.com/keyphact/pgoapi.git (see updated readme).

Second, I was getting this exception because the login process didn't work (due to a credential input typo). I found this out by adding in some debugging and then I saw this:
ERROR:pgoapi.pgoapi:Login process failed: Invalid Google Username/password


Here's how to spit out the logging to the console:
```
import logging,sys
...
logging.basicConfig(stream=sys.stderr, level=logging.DEBUG)
```
